models/java/rule-factory/pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.mobilitydata.gbfs</groupId>
     <artifactId>rule-factory</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.2</version>
```

Comment:
Should we have all the jar versions follow each other? i.e. everything at 1.0.3
you could have the child pom.xml inherit the version from the parent maybe

Reply:
Only published pom needs version updates. Updating the version here just adds work since you then also have to update its reference in the other pom. Since it's not actually used anywhere else and it's always in sync (same repo) it's just wasted work.

Reply:
So you mean it was not necessary to update the rule-factory version?
So maybe in that case we should keep it a 1.0.0.

Reply:
Indeed, I kept it always at 1.0.0. Same with parent pom.

Reply:
It's probably not that clear, but rule-factory is not a runtime or even compile time dependency. It's used by the code generator and the generate sources phase only. The library user never interacts with it.

Reply:
If we want the versions to be consistent in all files, we need to add a parent section in the child poms. The full change is:
- Add parent section in child poms:
```
    <parent>
        <groupId>org.mobilitydata.gbfs</groupId>
        <artifactId>gbfs-java-model-parent</artifactId>
        <version>1.0.2</version>
    </parent>
```
- Delete the version tag in the child poms
- Use versions plugin to bump versions at once: `mvn versions:set -DnewVersion=x.y.x`

Reply:
There's a good reason I didn't do it that way - that would result in all the poms being published to maven central, when they are not needed. Only the pom of gbfs-java-model should be published.